lib/datacrafts_io_skeleton.rb

```diff
@@ -20,12 +20,18 @@ class Composer < ::Thor
     include OptionsExtractor
     include Helpers
 
-    desc "create APP_NAME", "Create you own best Rails/JS app"
+    desc "create APP_NAME", <<~DESC
+      Create your own best Rails/JS app.
+      If you have some issues with creation please use VERBOSE=true
+    DESC
 
     def create(app_name)
-      say "Skeleton in creating your best Rails/JS application \n"
+      say <<~DESC
+        Skeleton in creating your best Rails/JS application.
```

Comment:
It should be "Skeleton `is` creating your best Rails/JS application.", shouldn't it?